app/controllers/emancipations_controller.rb

```diff
@@ -0,0 +1,83 @@
+class EmancipationsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :require_organization!
+
+  rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
```

Comment:
You can remove this since this line is already in `ApplicationController` which the `EmancipationsController` inherits from.

Reply:
`user_not_authorized` in that file makes it so the `save` endpoint renders a json error response and not an html one.

Reply:
Ah, I see now you are using a different method name. You can also remove this line, and just rename your `user_not_authorized` method to `not_authorized`, and then the `not_authorized` method in this controller will override the one in the application controller.

Reply:
ah ok